src/pages/AuthPage/AuthPageStyles.js

```diff
@@ -13,4 +13,12 @@ export const StyledHeroContainer = styled.main`
   justify-content: space-evenly;
   flex-grow: 1;
   width: 100%;
+
+  @media screen and (max-width: 900px) {
```

Comment:
follow mobile first approach

Reply:
Setting padding and margin as base without this media query would require me to reset padding and margin at line 22.
So retaining this line may be better as it aligns with DRY principles.